src/store/metadata/stub/content/editions/adult-obesity-prevalence-aged-18-years-and-over_2023-05.json

```diff
@@ -0,0 +1,155 @@
+    "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions",
+    "@type": "hydra:Collection",
+    "title": "Adult obesity prevalence (aged 18 years and over)",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over",
+        "identifier": "2023-05",
+        "title": "Adult obesity prevalence (aged 18 years and over)",
+        "summary": "Percentage of adults aged 18 and over classified as living with obesity, England, 16/11/2015-15/11/2016 to 16/11/2021-15/11/2022.",
+        "description": "Percentage of adults aged 18 and over classified as living with obesity, in England, for 16/11/2015-15/11/2016 to 16/11/2021-15/11/2022.\n\nCoverage\nEngland\n\nGeography definition\nLocal Authority Districts and Unitary Authorities,\nCounties,\nRegions,\nCombined Authorities,\nNation (England)\n\nSource\nEngland: [Office for Health Improvement and Disparities]\n\nSource Notes and Caveats\nAdults are defined as obese if their body mass index (BMI) is greater than or equal to 30kg/m²",
+        "issued": "2023-05-17T12:41:00Z",
+        "modified": "2023-12-13T09:01:00Z",
+        "next_release": "1900-01-01T00:00:00Z",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/office-for-health-improvement-and-disparities",
+        "contact_point": {
+          "name": "Office for Health Improvement and Disparities",
+          "email": "mailto:dhsc.publicenquiries@dhsc.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/health-and-social-care"
+        ],
+        "frequency": "irregular",
+        "keywords": [
+          "obesity",
+          "health",
+          "subnational",
+          "community",
+          "county"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "spatial_resolution":["E0", "E1", "E9"],
+      "spatial_coverage": "E92000001",
+      "temporal_resolution": ["P1Y"],
+      "temporal_coverage": {
+        "start": "2015-11-16T00:00:00Z",
+        "end": "2021-11-16T00:00:00Z"
+       },
+        "versions_url": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions/1",
+            "issued": "2023-05-17T12:41:00Z"
+          }
+        ],
+        "table_schema": {
+          "columns": [
+            {
+                "name": "period",
+                "datatype": "string",
+                "titles": "gregorian-interval",
```

Comment:
Please change to `"Gregorian Time Interval"`. Titles are usually title case.

 `"from_template": "gregorian-interval" `on the json file on family-ess repo only indicates that the gregorian-interval template was used in the csvcubed transform.
Here is a reference to check the titles:
https://beta.gss-data.org.uk/cube/explore?uri=https%3A%2F%2Fgss-cogs.github.io%2Ffamily-ess%2Fout%2Fadult-obesity-prevalence-aged-18-years-and-over%2Fadult-obesity-prevalence-aged-18-years-and-over%2Fadult-obesity-prevalence-aged-18-years-and-over.csv%23dataset-catalog-entry